docs/environment.md

```diff
@@ -17,53 +17,27 @@ In order to build the container follow the steps below:
    git clone https://github.com/fobnail/fobnail-sdk.git
    ```
 3. Go to fobnail SDK directory: `cd fobnail-sdk`
-4. Execute `./build.sh`. It will build the container from the most recent
-   stable software components described in
-   [tools-versions file](https://github.com/fobnail/fobnail-sdk/blob/main/tools-versions).
-   Components that are used in the process:
-   - NRF commandline tools: v10.14.0
-   - Zephyr SDK: v0.13.0
-   - Zephyr RTOS: v2.6.0
-   - Segger JLink: V7.54a
-   - cmake: 3.21.2
+4. To use latest container checkout `rust` branch: `git checkout rust`
+5. Execute `docker build . -f Dockerfile.rust -t 3mdeb/fobnail-rust-sdk`. It
+   will build the container with following software available:
+   - Rust 1.55.0
+   - Cargo-embed: always the latest version available from Cargo registry
 
 You will need fobnail-sdk to proceed with any work. The process will take a
 while to build the container. If we have built the container, time to verify
 it. We will build a sample application from the fobnail directory using the
 freshly built docker container. Follow the steps below to test the container:
 
-1. `git clone https://github.com/fobnail/fobnail.git`
-2. `cd fobnail`
-3. `git submodule update --init --checkout`
-4. `./run-container.sh` This will launch the container. It will take a moment
-   to initialize when invoke for the first time in given directory.
-5. Build a blinky application:
+1. `git clone https://github.com/fobnail/nrf-hal`
+2. `cd nrf-hal`
+3. Start container: `./run-container.sh`
```

Comment:
I would specify a git branch where `run-container.sh` is located. There is no such file in `master` branch

Reply:
Done